tools/license/get_dep_licenses.go

```diff
@@ -184,12 +270,65 @@ func main() {
 		linfo.path = lp
 		linfo.url = pathToURL(lp)
 		licenses = append(licenses, linfo)
+		ltypeStr := linfo.licenseTypeString
 		if linfo.exact {
-			licenseTypes[linfo.licenseTypeString] = append(licenseTypes[linfo.licenseTypeString], p)
+			licenseTypes[ltypeStr] = append(licenseTypes[ltypeStr], p)
 			exact = append(exact, linfo)
 		} else {
 			inexact = append(inexact, linfo)
 		}
+
+		fmt.Printf("Checking %s\n", linfo.packageName)
+		lt, ok := strToLicenseType[ltypeStr]
+		switch {
+		// No license was found by licensee.
+		case ltypeStr == "":
+			missingList = append(missingList, linfo.packageName)
+		// License was found but not in a definite category.
+		case !ok:
+			if !knownUnknownLicenses[linfo.packageName] {
```

Comment:
Can "KnownUnknown" be moved to the same "strToLicenseType" map and avoid this?

Reply:
No, because this is mapping a package name, not a license string.